src/wlapplication.cc

```diff
@@ -980,7 +994,14 @@ void WLApplication::parse_commandline(int const argc, char const* const* const a
 
 		// Are we looking at an option at all?
 		if (opt.compare(0, 2, "--")) {
-			throw ParameterError();
+			if (argc == 2) {
```

Comment:
Maybe also check if it ends with a .wgf extension here? I'm not sure.

Reply:
I omitted additional checks here because you will get a GUI message, if it isn't a wgf ("incompatible file").

Reply:
I was thinking of command line use. Typing `./widelands badargument` will attempt to load `badargument` instead of printing the usage information. But maybe that's fine.

Speaking of usage information (the one printed with `--help`), do add information there about this behavior.

Reply:
Ah, ok. The argument now requires a wgf extension.